src/api/.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2019-03-15 06:48:15 +0000 using RuboCop version 0.65.0.
+# on 2019-03-20 15:24:35 +0000 using RuboCop version 0.65.0.
```

Comment:
why it is needed?

Reply:
You can read the commit. I will paste it here:

With the addition of code in file `src/api/lib/tasks/dev.rake`, cop
Metrics/BlockLength complained about the length of the `dev` block.

Recreating the RuboCop TODO file with the
`dev:lint:rubocop:auto_gen_config:rails` task increments the length limit,
and then RuboCop doesn't complain.

Reply:
Does this Metrics/BlockLength make sense at *any* point?

Reply:
I would consider an iterator, like `map`, that contains a lot of code a code smell. For such cases the cop makes IMO sense.

We could do what rubocop [does](https://github.com/rubocop-hq/rubocop/commit/157b947473e32e6e79bedfa57798fdf9fc6a0650) and ignore this in case of rake files and gem specs...

Reply:
But the limit is 457 lines atm - that's not a code smell, that's a code toilet

Reply:
You were asking about the general sense of  Metrics/BlockLength:wink:

The interesting part is what maximum we would need after ignoring rake tasks and alike, as I proposed.

Reply:
@eduardoj I read the commit, but you shouldn't introduce new offenses... if this cop doesn't make sense for this file move it to `.rubocop.yml` (with a comment with the reason). If it is in the todo is because you plan to fix it and then you should fix it.

Reply:
I removed the commit that recreated the RuboCop TODO file, and I created one commit excluding the Metrics/BlockLength cop for `*.rake` files, as @bgeuken suggested. Please @Ana06 and @coolo, have a look again.